### DISCUSSION TITLE
[QUESTION] - How does everything comes together?

### DISCUSSION THREAD

**Discussion by mprather** [opencaesar/platform#14] [Q&A]
## Description
What is the overall workflow of the system?

What are the user artifacts that need to be created? How does this info then get transformed and utilized? What are the different stages and what tools manage each stage?

**Comment by melaasar** [comment]
I am working two new sections of the [OML spec](http://www.opencaesar.io/oml/), called Getting Started and Language Referene. They should shed light on your questions above.

However, in a nutshell, you can get started on a OML project by cloning the [oml-template](https://github.com/opencaesar/oml-template), and then authoring your vocabulary bundle (from scratch or by extending existing ones), and then using it to create a description bundle where you describe your system. Along the way, you can invoke analysis tools on your ontologies, to check their consistency, query them, transform them, etc.

The docs planned will walk a user through how to do these activities step by step.
